doc/rst/usingchapel/chplenv.rst

```diff
@@ -687,6 +686,9 @@ CHPL_LLVM
        system         find a compatible LLVM in system libraries;
                       note: the LLVM must be a version supported by Chapel
        none           do not support llvm/clang-related features
+       unset          indicates that no reasonable default has been
+                      inferred, requiring the user to intentionally select
+                      another option
```

Comment:
@mppf / @daviditen (upon returning): while working on CHANGES.md today, I noticed that `unset` was not described in this table which seemed asymmetrical to other cases to me (I tend to think of the tables as listing all of the options; I realize this case is a little asymmetrical in that we'd never expect anyone to purposefully set it).  But I wanted to make sure that this wasn't left off on purpose for reasons I wasn't picking up on.  Thanks.

Reply:
FWIW, I think it's fine and reasonable to add it to the table